src/Targets/Microsoft.VisualBasic.DesignTime.targets

```diff
@@ -31,21 +31,32 @@
       </PropertyPageSchema>
 
      <ProjectCapability Include="VB;Managed"/>
+  </ItemGroup>
+
+  <!-- Targets -->
+
+  <!-- Returns Vbc command-line arguments for the language service -->
+  <Target Name="CompileDesignTime"
+          Returns="@(_CompilerCommandLineArgs)"
+          DependsOnTargets="_CheckCompileDesignTimePrerequisite;Compile"
+          Condition="'$(IsCrossTargetingBuild)' != 'true'"
+        >
+
+    <!-- VB Framework projects needs core library references. In the case of Command line scenario, the vbc injects this reference. -->
+    <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
```

Comment:
@srivatsn Before adding the reference we check if the path exists, which would cover the `FrameworkPathOverride` check. This would also cover us from cases in the future, when we decide to set `FrameworkPathOverride`